__tests__/app.test.js

```diff
@@ -48,4 +49,34 @@ describe('/api (getting all endpoints)', () => {
         });
     });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('making sure that its responding with an article with the desired id', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then((article) => {
+            expect(article.body.article.article_id).toBe(2);
+        });
+    });
+
+    test('making sure that its responding with an article with the desired id when tested multipul times', () => {
+        return request(app)
+        .get('/api/articles/5')
+        .expect(200)
+        .then((article) => {
+            expect(article.body.article.article_id).toBe(5);
+        });
+    });
+
+    test('making sure that its when it fails it give a special error', () => {
```

Comment:
what kind of fail is happening here? from reading the test description we should know what kind of request we are expecting to fail. try to also have the expected status code in your test descriptions: e.g. "400 - invalid article_id" and "404: valid but non-existent article_id"